autoload/vader/window.vim

```diff
@@ -38,13 +38,16 @@ function! s:switch_to_workbench()
   execute 'b' s:workbench_bfr
 endfunction
 
-function! vader#window#open()
+function! vader#window#close()
   silent! bd \[Vader\]
-  silent! bd \[Vader-workbench\]
+  silent! bw! \[Vader-workbench\]
```

Comment:
I can confirm that vader opens the workbench buffer without Given block and has no problem closing it even when it's modified. See:

```
Execute:
  put ='foobar'
  redraw!
  normal! gs

Expect:

  foobar

Execute:
  put ='baz'
  redraw!
  normal! gs

Expect:

  baz
```

So I don't know what the problem is. Can you give me with a minimal vader scenario that doesn't work as expected?